src/components/SortButton/index.tsx

```diff
@@ -0,0 +1,50 @@
+import { useHistory, useLocation } from 'react-router-dom'
+import { ReactComponent as SortIcon } from '../../assets/sort_icon.svg'
+import styles from './styles.module.scss'
+
+enum SortOrder {
+  Asc = 'asc',
+  Desc = 'desc',
+}
+
+/*
+ * REFACTOR: could be refactored for https://github.com/Magickbase/ckb-explorer-frontend/pull/8#discussion_r1267484265
+ */
+const SortButton: React.FC<{
+  field: string
+}> = ({ field }) => {
+  const { push } = useHistory()
+  const { search, pathname } = useLocation()
+
+  const query = new URLSearchParams(search)
+
+  const [sortKey, sortOrder] = query.get('sort')?.split('.') ?? []
+
+  const isActive = sortKey === field
+
+  const handleClick = () => {
+    const shouldAsc = isActive && sortOrder === SortOrder.Desc
+    push(
+      `${pathname}?${new URLSearchParams({
+        ...Object.fromEntries(query),
+        sort: shouldAsc ? `${field}.${SortOrder.Asc}` : `${field}.${SortOrder.Desc}`,
+      })}`,
+    )
+  }
+
+  return (
+    <button
+      type="button"
+      className={styles.container}
+      data-is-asc={isActive && sortOrder === SortOrder.Asc}
+      data-is-desc={isActive && sortOrder === SortOrder.Desc}
```

Comment:
optional: use [classnames](https://www.npmjs.com/package/classnames) to make it more simple

Reply:
The attributes are simplified to `data-order=asc | desc | null` in https://github.com/Magickbase/ckb-explorer-frontend/pull/8/commits/96166d9ee09fde28e21174383dd3cb467d174d7e